packages/blocks/src/__internal__/utils/query.ts

```diff
@@ -931,29 +931,47 @@ export function getDropRectByPoint(
   point: Point,
   model: BaseBlockModel,
   element: Element
-) {
+): {
+  rect: DOMRect;
+  flag: number;
+} {
   const result = {
     rect: getRectByBlockElement(element),
-    // 0: others, 1: empty database
+    // 0: others, 1: empty database, 2: database
```

Comment:
I think it's better to use a typed `type` field (or enums) instead of hard-coded integers.